styles/utils/mixins.scss

```diff
@@ -0,0 +1,38 @@
+@mixin for-mobile-big() {
+  @media (min-width: 744px) {
+    @content;
+  }
+}
+
+@mixin for-desktop() {
+  @media (min-width: 1260px) {
```

Comment:
The media query for desktop devices seems to have a min-width of 1260px. Common breakpoints for tablets are usually 768px, and 992px or 1200px for desktops. Consider adjusting this value to cater to these devices better.